changelogs/fragments/2060-zos_mount-skip_fsumf168_from_df.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - zos_mount - Added handling to the df call, so that FSUMF168 are ignored.
```

Comment:
We try to stick to the ansible development process for change logs https://docs.ansible.com/ansible/latest/community/development_process.html . In this case this is the format we use 
```
Fixes that resolve issues. SHOULD not be used for minor enhancements (use minor_change instead). Use past tense to describe the problem and present tense to describe the fix.
```

And since this bugfix will be client-facing it should be less technical more like "In some cases, df command would raise error FSUMF168 as a warning for certain cases(can add more details for those cases) and the module failed. Fix now handles this warning message gracefully and continues with the operation."

Reply:
Your comments make perfect sense for these cases :
```
FSUMF168 return in stderror means that the mount dataset wouldn't resolve.
        # While this shows a catalog or volume issue, it should not impact our search for an existing mount
```